vendor/bitcoin_support/src/pubkey.rs

```diff
@@ -51,6 +51,7 @@ impl From<Hash160> for PubkeyHash {
     }
 }
 
+#[allow(clippy::fallible_impl_from)]
```

Comment:
❌ I suggest we open an issue to track this TODO

Reply:
tracked in #629